### ISSUE TITLE
Cache server exit with error randomly

### ISSUE THREAD

**Issue by Jayatubi** [Unity-Technologies/unity-cache-server#84]
After the Cache server update to 6.2.1 it would exit unexpected in serveral minutes. Here is the error log:

```
/usr/lib/node_modules/unity-cache-server/lib/cluster_messages.js:24
.then(result => {
^
TypeError: Cannot read property 'then' of undefined
at EventEmitter.cluster.on (/usr/lib/node_modules/unity-cache-server/lib/cluster_messages.js:24:21)
at emitThree (events.js:136:13)
at EventEmitter.emit (events.js:217:7)
at Worker.<anonymous> (internal/cluster/master.js:172:13)
at emitTwo (events.js:126:13)
at Worker.emit (events.js:214:7)
at ChildProcess.Worker.process.on (internal/cluster/worker.js:42:12)
at emitTwo (events.js:126:13)
at ChildProcess.emit (events.js:214:7)
at emit (internal/child_process.js:772:12)
```

**Comment by stephen-palmer** [comment]
thanks - just reproduced it. I'll get a fix in asap. For now, just disable clustering (or downgrade back to 6.1.2)

**Comment by stephen-palmer** [comment]
Fixed in release 6.2.3
